cassandra-framework/src/main/java/io/mesosphere/mesos/frameworks/cassandra/Main.java

```diff
@@ -112,15 +112,18 @@ private static int _main() throws UnknownHostException {
             throw new IllegalArgumentException("number of nodes (" + executorCount + ") and/or number of seeds (" + seedCount + ") invalid");
         }
 
+        CassandraFrameworkProtos.CassandraConfigRole.Builder defaultConfigRole = CassandraFrameworkProtos.CassandraConfigRole.newBuilder()
+            .setCassandraVersion(cassandraVersion)
+            .setCpuCores(resourceCpuCores)
+            .setDiskMb(resourceDiskMegabytes)
+            .setNumberOfNodes(executorCount)
+            .setNumberOfSeeds(seedCount)
+            .setMemMb(resourceMemoryMegabytes);
+        CassandraCluster.fillConfigRoleGaps(defaultConfigRole);
```

Comment:
Defining the default role should be done in the default `Supplier` in [`PersistedCassandraFrameworkConfiguration`](https://github.com/mesosphere/cassandra-mesos/blob/rewrite/cassandra-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/PersistedCassandraFrameworkConfiguration.java#L46) rather than calling it in Main